meson.build

```diff
@@ -0,0 +1,41 @@
+project(
+  'mc_lib',
+  ['cpp', 'cython'],
+  default_options : ['warning_level=3'],
+  version: '0.3',
+  license: 'BSD-2',
+  meson_version: '>= 0.54',
+)
+
+if build_machine.system() == 'darwin'
+    add_global_arguments('-std=c++17', language : 'cpp')
+    message('found OS X, add special argument to compiler')
+endif
+
+
+if meson.backend() != 'ninja'
+  error('MESON_SKIP_TEST: Ninja backend required')
```

Comment:
`MESON_SKIP_TEST` is a leftover from somewhere else, this should be something like "Build error:`

Reply:
Or left out completely. Same for `MESON_BUILD_FAILED` below